### ISSUE TITLE
[BUG] Large video file not uploading in iOS app

### ISSUE THREAD

**Issue by wtrdk** [immich-app/immich#581]
**Describe the bug**
When uploading a large video file (6.52GB) using the iOS app it hangs at 3%

**System**
- Phone OS [iOS]: `16.0 beta`
- Server Version: `1.26.0`
- Mobile App Version: `1.26.0 build 51`

**Additional context**
The upload of this specific file (see screenshots) never passes 3%. All other uploads are processed correctly.

![IMG_7931](https://user-images.githubusercontent.com/4569244/188405502-ad394df6-d81f-43f4-a61a-ae897b5b7c6d.jpeg)
![IMG_7904](https://user-images.githubusercontent.com/4569244/188405506-3df2a889-ed25-4a51-9723-c3ad87e6244a.jpeg)

**Comment by wtrdk** [comment]
Okay, well... Browsing the issues and feature requests I bumped into this:
https://github.com/immich-app/immich/issues/567
which made me realise my Nginx Proxy Manager could be the issue.

By adding this to the Advanced tab of my Immich configuration, I got the upload to work!
`	proxy_set_header Host $host;
	proxy_set_header X-Forwarded-Proto $scheme;
	proxy_set_header X-Real-IP $remote_addr;
	proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
	proxy_max_temp_file_size 16384m;
	client_max_body_size 0;
`

So I hop this helps anyone else facing this issue!
